terraform/ec2-examples/distributed-ml-training/README.md

```diff
@@ -0,0 +1,74 @@
+# ECS machine learning distributed training
+
+This solution blueprint creates the infrastructure to run distributed training jobs using a [Ray cluster](https://docs.ray.io/en/latest/cluster/getting-started.html) and [PyTorch](https://pytorch.org/). The Ray head node runs on a m5.xlarge instance, while the 2 workers run on g5.12xlarge instances.
+
+![Solution architecture](docs/architecture.png)
+
+## Cost warning!
+
+By default, this blueprint uses g5.12xlarge (with 4 GPUs) to showcase multi-GPU and multi-node distributed training, but **can increase costs considerably over time**. You can modify this blueprint to use g5.xlarge (with one GPU) instead from the local variable **instance_type_workers** - if you change the instance type, you need to also modify the worker task definition to use 1 GPU instead of 4 (see **resource_requirements** and container command parameter **--num-gpus**) and the example training script outline below (see **num_workers** parameter)
```

Comment:
If we can leverage g5.xlarge, why don't we use g5.xlarge rather than 12xlarge for giving customer's convenience?

Reply:
The main reason of using 12xlarge is to showcase multi-gpu but I agree that using xlarge is more convenient. The blueprint now uses g5.xlarge